### DISCUSSION TITLE
Galaxy brain achievement

### DISCUSSION THREAD

**Discussion by noxs1d** [drknzz/GitHub-Achievements#281] [General]
Guys can u help me to achieve it. Just 2 replies
<img width="148" alt="GalaxyBrain" src="https://github.com/drknzz/GitHub-Achievements/assets/117533743/ff2e107a-b9cf-4dbc-9c1d-988ed571b6a6">Thanks A lot

**Comment by the-coder-o** [comment]
Helloooo thank you

**Reply by spidybroz** [reply]
## ⭐ Star & 🍴 Fork the Repo <3

If you like this project, please **star** ⭐ the repo — I’ll star yours back!  
Feel free to **fork** 🍴 it, contribute, or share it with others 🚀 

### 🔗 Links
- **GitHub:** https://github.com/maiz-an/DevTunnel-CLI  
- **Website:** https://devtunnel-cli.vercel.app

#### [maiz-an](https://github.com/maiz-an)

**Comment by kalungia** [comment]
Here you go, mark it the answer!

**Comment by the-coder-o** [comment]
qwwwwooowww

**Reply by AmaruVL** [reply]
Alright, no problem

**Comment by doriyan13** [comment]
i think you need to get your answer being accepted on others discussion 😃

**Reply by noxs1d** [reply]
> i think you need to get your answer being accepted on others discussion 😃 

Yes

**Reply by kalungia** [reply]
Yah

**Reply by doriyan13** [reply]
Can you mark my answer I want the badge also hehe ;)

**Reply by noxs1d** [reply]
> Can you mark my answer I want the badge also hehe ;)

I cant, it's not my discussion

**Reply by doriyan13** [reply]
Ohhh I c

**Comment by devIgor1** [comment]
Heyy, thanks a lot!

**Comment by melekbadreddine** [comment]
i want to earn it as well, kindly mark my answer.

**Comment by npmRunTheWorld** [comment]
Need that galaxy brain thanks!

**Comment by ojasvinaik13** [comment]
I want this badge too, help me out!

**Comment by gaurav0401** [comment]
hello

**Comment by spidybroz** [comment]
## ⭐ Star & 🍴 Fork the Repo <3

If you like this project, please **star** ⭐ the repo — I’ll star yours back!  
Feel free to **fork** 🍴 it, contribute, or share it with others 🚀 

### 🔗 Links
- **GitHub:** https://github.com/maiz-an/DevTunnel-CLI  
- **Website:** https://devtunnel-cli.vercel.app

#### [maiz-an](https://github.com/maiz-an)
